VENDORING.md

```diff
@@ -0,0 +1,43 @@
+# Vendoring policies
+
+This document outlines recommended Vendoring policies for Docker repositories.
+(Example, libnetwork is a Docker repo and logrus is not.)
+
+## Vendoring using tags
+
+Commit ID based vendoring provides little/no information about the updates
+vendored. To fix this, vendors will now require that repositories use annotated
+tags to snapshot commits.
+
+Each tag should:
+- Follow Semantic Versioning rules (refer to section on "Semantic Versioning")
+- Have a corresponding entry in the repo's CHANGELOG.md that describes changes
+since the previous update.
+
+Each repo should:
+- Have a mandatory CHANGELOG.md at the root of the repo.
+
+The goal here is for consuming repos to be able to use the tag version and
```

Comment:
This is subjective. Ideally, we want the vendor code to be unit tested in their repos and only integration tests in the docker/docker repo. Anything that needs unit testing of vendored code on the main repo is an exception to the rule and is not a policy enforcement (i.e doesnt belong in this doc)